<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgAnimation</class>
 <widget class="QDialog" name="CDlgAnimation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttons">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeSequence">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>64</width>
     <height>201</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btAdd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Add frame</string>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btRemove">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete frame</string>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeIcons">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>64</width>
     <height>201</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>64</number>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btCopyAll">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Copy all the frame.</string>
   </property>
   <property name="text">
    <string>Copy All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btClearAll">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Clear frame sequence.</string>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>The available images are on the right. Use the appropriate buttons to copy them to the box on the left and create the animation sequence.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sDescription">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>blah blah blah blah blah blah blah blah  blah blah blah blah blah blah blah blah  blah blah blah blah blah blah blah blah  blah blah blah blah blah blah blah blah </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cRepete">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Repeate animation sequence</string>
   </property>
   <property name="text">
    <string>Repeated</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>CDlgAnimation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>CDlgAnimation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
